testsrc/test262.properties

```diff
@@ -3304,9 +3568,7 @@ language/statements/for-of
     ! dstr-obj-id-init-yield-ident-valid.js
     ! dstr-obj-id-put-const.js
     ! dstr-obj-id-put-let.js
-    ! dstr-obj-id-put-unresolvable-no-strict.js
-    ! dstr-obj-id-put-unresolvable-strict.js
-    ! dstr-obj-id-simple-no-strict.js
+    ! dstr-obj-id-simple-strict.js
```

Comment:
looks great, seems like many more tests passing. Any idea about this one - why was this passing in the past?

Reply:
https://github.com/tc39/test262/blob/041da54c02ae7d17edb8c134ab7691c4f643bafe/test/language/statements/for-of/dstr-obj-id-simple-strict.js
Previously, it was an error because shorthand was not possible. Correctly, it should be an error because it cannot be assigned to `eval` in strict mode.